presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestGrantRevoke.java

```diff
@@ -37,55 +42,48 @@
      * "bob@presto" that has "jdbc_user: bob"
      * (all other values of the connection are same as that of the default "presto" connection).
     */
-    @Test(groups = {HIVE_CONNECTOR, AUTHORIZATION, PROFILE_SPECIFIC_TESTS})
-    public void testGrantRevoke()
+
+    @BeforeTestWithContext
+    public void setup()
     {
-        String tableName = "alice_owned_table";
-        QueryExecutor queryExecutorForAlice = connectToPresto("alice@presto");
-        QueryExecutor queryExecutorForBob = connectToPresto("bob@presto");
+        tableName = "alice_owned_table";
+        aliceExecutor = connectToPresto("alice@presto");
+        bobExecutor = connectToPresto("bob@presto");
 
-        queryExecutorForAlice.executeQuery(format("DROP TABLE IF EXISTS %s", tableName));
-        queryExecutorForAlice.executeQuery(format("CREATE TABLE %s(month bigint, day bigint)", tableName));
+        aliceExecutor.executeQuery(format("DROP TABLE IF EXISTS %s", tableName));
+        aliceExecutor.executeQuery(format("CREATE TABLE %s(month bigint, day bigint)", tableName));
 
-        assertThat(() -> queryExecutorForBob.executeQuery(format("SELECT * FROM %s", tableName))).
-                failsWithMessage(format("Access Denied: Cannot select from table default.%s", tableName));
-        assertThat(() -> queryExecutorForBob.executeQuery(format("INSERT INTO %s VALUES (3, 22)", tableName))).
-                failsWithMessage(format("Access Denied: Cannot insert into table default.%s", tableName));
+        assertAccessDeniedOnAllOperationsOnTable(bobExecutor, tableName);
```

Comment:
Both the tests in this class need to call this piece of code to make sure that bob doesn't have any privileges on the table to begin with. So I extracted it to this common `setup` method.